src/interfaces/IMorpho.sol

```diff
@@ -123,10 +123,10 @@ interface IMorpho {
     /// @dev Warning: The recipient can be the zero address.
     function setFee(MarketParams memory marketParams, uint256 newFee) external;
 
-    /// @notice Sets `newFeeRecipient` as recipient of the fee.
-    /// @dev Warning: The fee recipient can be set to the zero address.
-    /// @dev Warning: The fee to be accrued on each market won't belong to the old fee recipient after calling this
-    /// function.
+    /// @notice Sets `newFeeRecipient` as `feeRecipient` of the fee.
+    /// @dev Warning: If the fee recipient is set to the zero address, fees will accrue there and will be lost.
+    /// @dev Modifying the fee recipient will allow the new recipient to claim any pending fees not yet accrued. To
+    /// ensure that the current recipient receives all due fees, accrue interest manually prior to making any changes.
```

Comment:
Mentioning the use case of accruing interest really makes me wonder if we should make the `accrueInterest` function public

Reply:
I tend to think we should, it would allow integrators to have more certainty on their actions I think

Reply:
Let's discuss that separately, here is the [related thread](https://morpholabs.slack.com/archives/C02N7CZ088N/p1698164347788439)